frontend/server/src/main/java/com/amazonaws/ml/mms/util/logging/QLogLayout.java

```diff
@@ -14,27 +14,36 @@
 
 import com.amazonaws.ml.mms.metrics.Dimension;
 import com.amazonaws.ml.mms.metrics.Metric;
+import org.apache.log4j.Logger;
 import org.apache.log4j.PatternLayout;
 import org.apache.log4j.spi.LoggingEvent;
 
 public class QLogLayout extends PatternLayout {
+    private static final Logger logger = Logger.getLogger(QLogLayout.class);
 
     @Override
     public String format(LoggingEvent event) {
         Object eventMessage = event.getMessage();
         if (eventMessage == null) {
             return null;
         }
+        String marketPlace = "NONE";
+        try {
+            marketPlace = System.getenv("REALM");
+        } catch (Exception e) {
+            logger.info("REALM env variable not set or could not be accessed");
+        }
         if (eventMessage instanceof Metric) {
             StringBuilder stringBuilder = new StringBuilder();
             Metric metric = (Metric) eventMessage;
             stringBuilder.append("HostName=").append(metric.getHostName());
             if (metric.getRequestId() != null) {
                 stringBuilder.append("\nRequestId=").append(metric.getRequestId());
             }
-            stringBuilder.append("\nTimeStamp=").append(metric.getTimestamp());
+            stringBuilder.append("\nMarketPlace=").append(marketPlace);
+            stringBuilder.append("\nStartTime=").append(metric.getTimestamp());
             stringBuilder
-                    .append("\nMetrics=")
+                    .append("\nProgram=MXNetModelServer\nMetrics=")
```

Comment:
Should this be called the package name? I am not sure what the expectation here is.. Usually package names are used for Program names.

Reply:
Sure I can do a renaming for that